backend/eveonline/models.py

```diff
@@ -91,6 +91,27 @@ def __str__(self):
         return str(self.name)
 
 
+class EveCharacterTag(models.Model):
+    """EveCharacter Tag model"""
+
+    character = models.ForeignKey(EveCharacter, on_delete=models.CASCADE)
+    choices: tuple = (
+        ("DREAD", "1"),
```

Comment:
The right side is the "display side" so these should be user-friendly slugs, e.g "Dread Pilot"